pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -112,14 +131,19 @@ var (
 
 	caRSAKeySize = env.RegisterIntVar("CITADEL_SELF_SIGNED_CA_RSA_KEY_SIZE", 2048,
 		"Specify the RSA key size to use for self-signed Istio CA certificates.")
-)
 
-type CAOptions struct {
-	// domain to use in SPIFFE identity URLs
-	TrustDomain    string
-	Namespace      string
-	Authenticators []authenticate.Authenticator
-}
+	//TODO: Likely to be removed and added to mesh config
+	externalCA = env.RegisterStringVar("EXTERNAL_CA", "",
```

Comment:
Add the valid values

Reply:
will do

Reply:
Why do we need this ? If we do ( and I don't think so) - should be EXERNAL_CA_TYPE.

But it is sufficient to check if K8S_SIGNER is set to determine if we use k8s, no need for redundant fields

Reply:
This commit was also partly to integrate with external CA's that integrated with Istiod using the ISTIO CA GRPC API. Hence I thought two flags were appropriate -> One to indicate that Istiod was to function as an RA and NOT a CA (externalCA), the other to specify the configuration for integrating with an external K8s CA (i.e signerName)